app/presenters/hyrax/presents_attributes.rb

```diff
@@ -28,6 +28,18 @@ def permission_badge_class
       PermissionBadge
     end
 
+    def workflow_badge
```

Comment:
Why is this in this module if it can't be shared? Why not just on the presenter?